src/SFA.DAS.Campaign.UnitTests/Infrastructure/Api/Queries/WhenGettingAHub.cs

```diff
@@ -17,49 +16,111 @@ namespace SFA.DAS.Campaign.UnitTests.Infrastructure.Api.Queries
 {
     public class WhenGettingAHub
     {
-        [Test, RecursiveMoqAutoData]
-        public async Task Then_The_Api_Is_Called_With_The_Valid_Request_Parameters_And_The_Hub_Is_Returned_From_The_Preview_Api_if_Config_And_Param_Set(
-            GetHubQuery query, Page<Hub> response, [Frozen] Mock<IOptions<CampaignConfiguration>> config, [Frozen] Mock<IApiClient> client, GetHubQueryHandler handler)
+        [Test]
+        [MoqInlineAutoData(true, true)]
+        [MoqInlineAutoData(true, false)]
+        [MoqInlineAutoData(false, true)]
+        [MoqInlineAutoData(false, false)]
+        public async Task AndForcePreviewIsTrue_ThenTheHubIsReturnedFromThePreviewAPI(
+            bool allowPreview, bool previewWanted, GetHubQuery query, Page<Hub> response, [Frozen] Mock<IOptions<CampaignConfiguration>> config, [Frozen] Mock<IApiClient> client, GetHubQueryHandler handler)
         {
-            SetupMockConfig(config);
-            query.Preview = true;
+            query.Preview = previewWanted;
+            SetupMockConfig(config, allowPreview, true);
             client.Setup(o => o.Get<Page<Hub>>(It.Is<GetHubPreviewRequest>(r => r.GetUrl == $"hub/preview/{query.Hub}"))).ReturnsAsync(response);
 
             var actual = await handler.Handle(query, CancellationToken.None);
 
-            client.Verify(o => o.Get<Page<Hub>>(It.Is<GetHubPreviewRequest>(r => r.GetUrl == $"hub/preview/{query.Hub}")), Times.Once);
-            actual.Should().NotBeNull();
-            actual.Page.Should().NotBeNull();
+            Assert.Multiple(() =>
+            {
+                client.Verify(o => o.Get<Page<Hub>>(It.Is<GetHubPreviewRequest>(r => r.GetUrl == $"hub/preview/{query.Hub}")), Times.Once);
+                actual.Should().NotBeNull();
+                actual.Page.Should().NotBeNull();
+            });
         }
 
-        [Test]
-        [RecursiveMoqInlineAutoData(true)]
-        [RecursiveMoqInlineAutoData(false)]
-        public async Task Then_The_Api_Is_Called_With_The_Valid_Request_Parameters_And_Preview_Is_Disabled_Then_The_Hub_Is_Returned_From_The_Api(
-            bool preview, GetHubQuery query, Page<Hub> response, [Frozen] Mock<IApiClient> client, [Frozen] Mock<IOptions<CampaignConfiguration>> config, GetHubQueryHandler handler)
+        [Test, MoqInlineAutoData(true, true)]
```

Comment:
The name of the test contains AndAllowPreviewIsTrue but that is an optional argument